command/main.go

```diff
@@ -264,3 +254,47 @@ func printCommand(w io.Writer, name string, cmdFn cli.CommandFactory) {
 	}
 	fmt.Fprintf(w, "    %s\t%s\n", name, cmd.Synopsis())
 }
+
+func generateCurlString(exitCode int, runOpts *RunOptions) int {
+	if exitCode == 0 {
+		fmt.Fprint(runOpts.Stderr, "Could not generate cURL command")
+		return 1
+	} else {
+		if api.LastOutputStringError == nil {
+			if exitCode == 127 {
+				// Usage, just pass it through
+				return exitCode
+			}
+			fmt.Fprint(runOpts.Stderr, "cURL command not set by API operation; run without -output-curl-string to see the generated error\n")
+			return exitCode
+		}
+		if api.LastOutputStringError.Error() != api.ErrOutputStringRequest {
+			runOpts.Stdout.Write([]byte(fmt.Sprintf("Error creating request string: %s\n", api.LastOutputStringError.Error())))
+			return 1
+		}
+		runOpts.Stdout.Write([]byte(fmt.Sprintf("%s\n", api.LastOutputStringError.CurlString())))
+		return 0
+	}
+}
+
+func generatePolicy(exitCode int, runOpts *RunOptions) int {
+	if exitCode == 0 {
+		fmt.Fprint(runOpts.Stderr, "Could not generate policy")
+		return 1
+	} else {
```

Comment:
The `else` isn't necessary here: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow